src/super_gradients/training/models/sg_module.py

```diff
@@ -64,5 +64,16 @@ class to implement.
 
         raise NotImplementedError
 
-    def replace_input_channels(self, in_channels: int, compute_new_weights_fn: Optional[Callable[[nn.Module, int], nn.Module]] = None):
-        raise NotImplementedError
+    def get_finetune_lr_dict(self, lr: float) -> Dict[str, float]:
+        """
+        Returns a dictionary, mapping lr to the unfrozen part of the network, in the same fashion as using initial_lr in trianing_params
+         when calling Trainer.train().
+        For example:
+            def get_finetune_lr_dict(self, lr: float) -> Dict[str, float]:
+                return {"default": 0, "head": lr}
+
+        :param lr: float, learning rate for the part of the network to be tuned.
+        :return: learning rate mapping that can be used by
+         super_gradients.training.utils.optimizer_utils.initialize_param_groups
+        """
+        raise NotImplementedError("Finetune is not implemented for this model, it is required to" " implement get_finetune_lr_dict.")
```

Comment:
Some code formatting issues: ` to" " imple`

Reply:
5321b2de50e9e66ff755d3663be919bdd50c112f fixed